language/tools/move-cli/src/lib.rs

```diff
@@ -92,6 +88,18 @@ pub struct MoveCLI {
 
 #[derive(StructOpt)]
 pub enum Command {
+    #[structopt(name = "package")]
+    Package {
+        /// Path to package. If none is supplied the current directory will be used.
+        #[structopt(long = "path", short = "p", global = true)]
+        path: Option<String>,
```

Comment:
Same as above, technically this could be a `PathBuf`. Up to you!

(There are several other options in this pull request that use `String` to refer to a file path. I'll stop commenting on each one.)

Reply:
Went through and updated things to use `PathBuf` and `Path` where appropriate instead of `String`.